src/types/api.ts

```diff
@@ -193,6 +232,77 @@ export interface operations {
       422: unknown
     }
   }
+  /** Get filterable incoming transfers */
+  incoming_transfers: {
+    parameters: {
+      path: {
+        chainId: string
+        address: string
+      }
+      query?: {
+        execution_date__gte?: string
+        execution_date__lte?: string
```

Comment:
If I compare these to the docs: https://safe.global/safe-client-gateway/docs/routes/transactions/filters/transfer/struct.TransferFilters.html

I think this should be a single underscore in the end?
```suggestion
        execution_date_gte?: string
        execution_date_lte?: string
```

Reply:
This is indeed confusing but they match the [transaction service parameters](https://safe-transaction.rinkeby.gnosis.io/) so the gateway can simply forward them on. We can hopefully add more of the parameters in the future and then need not adjust the name in the gateway.

Reply:
Oh you are right.
I think this field annotation(?) in here might be responsible for that.
https://github.com/safe-global/safe-client-gateway/blob/main/src/routes/transactions/filters/transfer.rs

I didn't really know what to compare all values with :D So I just checked the docs.